src/utils.js

```diff
@@ -230,7 +230,7 @@ export const getAllPaths = (routes: RouteShape | RouteShape[]): string[] => {
  * assumed that this is a 404 route. See the RR changelong for details:
  * https://github.com/rackt/react-router/blob/1.0.x/CHANGES.md#notfound-route
  */
-export const getAssetKey = (location: string): string => {
+export const getAssetKey = (location: string, indexPaths : boolean): string => {
```

Comment:
To remain consistent with the rest of the codebase please remove the space between a variable and it's flow type. i.e. `indexPaths: boolean`